### PR TITLE
sync flow with local secp256 k1 key

### PR DESCRIPTION
**Pull request by KannuSingh**

### CODE REVIEW

File: src/components/sections/local-signer.tsx [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+    const _userOpHash = await bundlerClient.sendUserOperation({
+      userOperation: userOp
+    })
+
+    let txReceipt: GetUserOperationReceiptReturnType | null = null
+    while (txReceipt == null)
+      txReceipt = await bundlerClient.getUserOperationReceipt({
+        hash: _userOpHash
+      })
```
Comment (lukaisailovic) [inline src/components/sections/local-signer.tsx]: How many requests are we sending here?

File: src/core/passkeys.ts [hunk 2]
```diff
@@ -1,139 +1,142 @@
...
-      'spki',
-      passkeyCredential.response.getPublicKey(),
-      {
-        name: 'ECDSA',
-        namedCurve: 'P-256',
-        hash: { name: 'SHA-256' },
+      user: {
+        displayName: 'Session key',
+        id: crypto.getRandomValues(new Uint8Array(32)),
+        name: 'session-key'
```
Comment (lukaisailovic) [inline src/core/passkeys.ts:47]: Why is this named session key?

File: src/components/sections/local-signer.tsx (hunk 1 above)
Comment (KannuSingh) [reply src/components/sections/local-signer.tsx]: only one to purchase donut, line58,

Comment (lukaisailovic) [reply src/components/sections/local-signer.tsx]: I'm asking about the while loop. It seems like its sending many network requests

Comment (KannuSingh) [reply src/components/sections/local-signer.tsx]: it is trying to get the receipt for the sent userOps until it get the response back, I should add a delay of few sec btw the requests, or 
best approach would be to use `waitForUserOperationReceipt`  method in permissionless which will do this internally.

Comment (lukaisailovic) [reply src/components/sections/local-signer.tsx]: Method in permissionless has a polling interval. Your code will send huge amount of requests until it gets a result. Its not the same. Lets stick with permissionless function.

Comment (KannuSingh) [reply src/components/sections/local-signer.tsx]: Yes yes, I meant do the same.

Comment (lukaisailovic) [reply src/components/sections/local-signer.tsx]: Alright. Lets amend and merge after

**Pull request merged**